### ISSUE TITLE
Create builtin Elixir compiler to be able to include it in projects

### ISSUE THREAD

**Issue by wende** [wende/elmchemy#6]

**Comment by Baransu** [comment]
What have to be done? What is the best way to implement that? I would like to do this 🙂

**Comment by wende** [comment]
#25 Has the multiple file compiler. 
All that needs to be done is a mix task, which does `elmchemy aggregate parsed_directory`
elmchemy is a binary that relies on elmchemy.js which is a node file.

So probably the easiest way would be to publish an npm package and hex package. Create a mix compiler then:

```
npm install -g elmchemy
mix archive.install git elmchemy/repo
```
and then add elmchemy to compiler

**Comment by wende** [comment]
This repo has a good example how to write a Mix compiler:
https://github.com/wende/scalixir

**Comment by Baransu** [comment]
Closed in #43
